android/impl.nix

```diff
@@ -95,7 +96,15 @@ in {
           ln -s "$applicationMk" "$out/jni/Application.mk"
 
         '' + concatStrings (builtins.map (arch:
-          let inherit (appSOs.${arch}) hsApp;
+          let inherit (appSOs.${arch}) hsApp nativeDeps;
```

Comment:
Indentation is confusing, should be
```nix
let 
   inherit ....;
   foo = ....;
   bar = ...;
```
so they line up.

Reply:
Fixed